### PR TITLE
Fetch UP info and add walletAddress to global state

### PR DESCRIPTION
**Pull request by skimaharvey**

### CODE REVIEW

File: components/overview/UserInfos.tsx [hunk 1]
```diff
@@ -1,13 +1,94 @@
...
+    `${address.slice(0, 6)}...${address.slice(-4)}`;
+
+  const lsp3FirstLink = (): void => {
+    if (lsp3JSON?.LSP3Profile.links[0]?.url) {
+      setFirstLink(lsp3JSON.LSP3Profile.links[0]);
+    }
+  };
+
+  useEffect(() => {
+    console.log('changed address: ', walletAddress, lsp3JSON);
```
Comment (Hugoo) [inline components/overview/UserInfos.tsx:51]: You can remove this

File: components/shared/SearchBar.tsx [hunk 2]
```diff
@@ -45,6 +48,7 @@ const SearchBar: React.FC = () => {
   const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
     e.preventDefault();
     if (validAddress) {
+      setWalletAddress(address);
```
Comment (Hugoo) [inline components/shared/SearchBar.tsx:51]: Why do it here and not on the other page after the push?

File: pages/overview/[id].tsx [hunk 3]
```diff
@@ -82,43 +91,51 @@ const AdressOverview: NextPage = () => {
     setLoading(false);
   };
 
-  const checkIsUp = async (address: string) => {
-    const isUPResult = await isUP(address, web3Provider);
-    if (!isUPResult) {
-      setLoading(false);
+  const checkIsUP = async (address: string) => {
+    console.log('checkIsUP');
```
Comment (Hugoo) [inline pages/overview/[id].tsx:95]: I suggest to add no console rule in eslint

We don't need these test log in production.

**Review by Hugoo — APPROVED** [review]

File: components/shared/SearchBar.tsx (hunk 2 above)
Comment (skimaharvey) [reply components/shared/SearchBar.tsx:51]: Actually I removed it already on my next commit. Kept this `setState` for login purpose.

File: pages/overview/[id].tsx (hunk 3 above)
Comment (skimaharvey) [reply pages/overview/[id].tsx:95]: sounds good. I will add it. Tend to forget them.

**Review by Hugoo — APPROVED** [review]

**Pull request merged**